File: c++/1365. How Many Numbers Are Smaller Than the Current Number/1365_for.cpp
class Solution {
public:
    vector<int> smallerNumbersThanCurrent(vector<int>& nums) { 
        vector<int> res;  //я создаю пустой список для будущего рез-та
        for (int i=0;i<nums.size();i++) //проверяю по индексам из nums
        {
            int count = 0; //завожу переменную которая будет считать сколько элементов больше
            for (int num=0;num<nums.size();num++) //теперь делаю for loop в самих nums по значениям
            {
                if (nums[i]>num) //если nums с индексом i больше чем значение num из nums
                    count += 1; //увеличим на единичку наш count
            }
            res.push_back(count); //в конце первого лупа мы добавим этот каунт в наш res
            //так и будут добавляться каунты
        }
        return res; //выведем список из всех каунтов
        
    }
};

File: python/1365. How Many Numbers Are Smaller Than the Current Number/1365_for.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        res = [] #making new result list
        for i in range(len(nums)): #and now go for loop in nums (checking by index)
            count = 0 #lets count how much counts we will have
            for num in nums: #now we will for looping in nums (checking by value)
                if nums[i] > num: #here is condition 
                    count+=1 #if its is we +=1 count
            res.append(count) #and add count for single index
        return res #return final list
